client/test/app/queue/substituteAppellant/tasks/utils.test.js

```diff
@@ -76,6 +76,139 @@ describe('utility functions for task manipulation', () => {
+    describe('when a ScheduleVeteranTask is selected', () => {
+      const selectedTaskTypes = ['ExampleTask', 'ScheduleHearingTask'];
+
+      const shouldDisables = [
+        'AssignHearingDispositionTask',
+        'ChangeHearingDispositionTask',
+        'EvidenceSubmissionWindowTask',
+        'TranscriptionTask'
+      ];
+
+      const shouldNotDisables = [
+        'ScheduleHearingTask',
+        'ExampleTask'
+      ];
+
+      it.each(shouldDisables)('should disable task type %s', (taskType) => {
+        expect(shouldDisableBasedOnTaskType(taskType, selectedTaskTypes)).toBe(true);
+      });
+
+      it.each(shouldNotDisables)('should not disable task type %s', (taskType) => {
+        expect(shouldDisableBasedOnTaskType(taskType, selectedTaskTypes)).toBe(false);
+      });
+    });
+
+    describe('when an EvidenceSubmissionWindow task is selected', () => {
+      const selectedTaskTypes = ['EvidenceSubmissionWindowTask'];
+
+      const shouldDisables = [
+        'ScheduleHearingTask',
+        'AssignHearingDispositionTask',
+        'ChangeHearingDispositionTask'
+      ];
+
+      const shouldNotDisables = [
+        'EvidenceSubmissionWindowTask',
+        'TranscriptionTask'
+      ];
+
+      // This block is the same for each of the three; there's an opportunity to refactor further!
+      it.each(shouldDisables)('should disable task type %s', (taskType) => {
+        expect(shouldDisableBasedOnTaskType(taskType, selectedTaskTypes)).toBe(true);
+      });
+
+      it.each(shouldNotDisables)('should not disable task type %s', (taskType) => {
+        expect(shouldDisableBasedOnTaskType(taskType, selectedTaskTypes)).toBe(false);
+      });
+    });
+
+    describe('when a TranscriptionTask is selected', () => {
+      const selectedTaskTypes = ['TranscriptionTask'];
+
+      const shouldDisables = [
+        'ScheduleHearingTask',
+        'AssignHearingDispositionTask',
+        'ChangeHearingDispositionTask'
+      ];
+
+      const shouldNotDisables = [
+        'TranscriptionTask',
+        'EvidenceSubmissionWindowTask'
+      ];
+
+      // This block is the same for each of the three; there's an opportunity to refactor further!
+      it.each(shouldDisables)('should disable task type %s', (taskType) => {
+        expect(shouldDisableBasedOnTaskType(taskType, selectedTaskTypes)).toBe(true);
+      });
+
+      it.each(shouldNotDisables)('should not disable task type %s', (taskType) => {
+        expect(shouldDisableBasedOnTaskType(taskType, selectedTaskTypes)).toBe(false);
+      });
+    });
+  });
+
+  describe('disabledTasksBasedOnSelections', () => {
+    const tasks = [
+      { taskId: 1, type: 'EvidenceSubmissionWindowTask' },
+      { taskId: 2, type: 'ScheduleHearingTask' },
+      { taskId: 3, type: 'AssignHearingDispositionTask' },
+      { taskId: 4, type: 'ChangeHearingDispositionTask' },
+      { taskId: 5, type: 'TranscriptionTask' }
+    ];
+
+    describe('when ScheduleVeteranTask is selected', () => {
```

Comment:
```suggestion
    describe('when ScheduleHearingTask is selected', () => {
```

D'oh! Will fix in the AM.